config/win32.gypi

```diff
@@ -25,6 +25,18 @@
 			'silence_warnings': 0,
 		},
 		
+		'include_dirs':
+		[
+			'$(foo)C:/WinDDK/7600.16385.1/inc/atl71',
+			'$(foo)C:/WinDDK/7600.16385.1/inc/mfc42',
+		],
+		
+		'library_dirs':
+		[
+			'$(foo)C:/WinDDK/7600.16385.1/lib/ATL/i386',
+			'$(foo)C:/WinDDK/7600.16385.1/lib/MFC/i386',
```

Comment:
I don't think those are necessary - so I'm trying removing them. The default setup of Vs2015/2017 seems to work fine without them.